styles.css

```diff
@@ -0,0 +1,739 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans&display=swap");
+
+@font-face {
+  font-family: "Lato";
+  src: url(./Lato2OFL/Lato-Medium.ttf);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.wrapper {
+  background-image: url(./Image/grey-bg.jpg);
+  width: 37px;
+}
+
+.nav-bar {
+  padding-top: 12px;
+  margin-left: 36px;
+  width: 375px;
+  height: 65px;
+}
+
+li {
+  list-style: none;
+}
+
+ul {
+  margin-top: 100px;
+  margin-left: 12px;
+}
+
+a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.nav-link {
+  font-family: "Lato", sans-serif;
+  line-height: 60px;
+  font-weight: 500;
+  color: #272a31;
+  font-size: 32px;
+  padding-left: 12px;
+}
+
+.nav-menu ul {
+  margin-top: 60px;
+}
+
+#btn-close {
+  position: absolute;
+  margin-left: 24px;
+  font-size: 32px;
+  font-weight: 500;
+  font-family: "Lato", sans-serif;
+  margin-top: 12px;
+  margin-bottom: 48px;
+  color: #ec5242;
+}
+
+#nav-menu {
+  display: none;
+  position: absolute;
+  height: 760px;
+  width: 375px;
+  text-align: left;
+  z-index: 99;
+  background-color: #d3d3d3;
+}
+
+.sec1 {
+  margin-top: 36px;
+  width: 345px;
+  height: 180px;
+  margin-left: 12px;
+}
+
+.sec1 h2 {
+  font-family: "Lato", sans-serif;
+  font-weight: 500;
+  font-size: 18px;
+  color: #ec5242;
+  margin-bottom: 0;
+}
+
+h1 {
+  font-family: "Lato", sans-serif;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 35px;
+  margin-left: 0;
+  margin-top: 0;
+  color: #ec5242;
+}
+
+.text {
+  font-family: "Lato", sans-serif;
+  font-weight: 600;
+  font-size: 12px;
+  color: #272a31;
+  width: 350px;
+  height: 120px;
+  margin-top: 0;
+  margin-bottom: 20px;
+  line-height: 25px;
+  padding: 12px;
+  margin-left: 14px;
+  border: 1px solid #fff;
+}
+
+.date {
+  font-family: "lato", sans-serif;
+  color: #272a31;
+  font-size: 28px;
+  font-weight: 700;
+  padding-left: 12px;
+  margin-bottom: 10px;
+}
+
+.venue {
+  font-family: "lato", sans-serif;
+  font-weight: 500;
+  color: #272a31;
+  text-align: left;
+  width: 295px;
+  height: 48px;
+  line-height: 22px;
+  margin-left: 12px;
+}
+
+/* Section 2 */
+.main-container {
+  padding: 12px;
+  height: 890px;
+  width: 375px;
+  background-image: url("./Image/black-bg.jpg");
+}
+
+.sec2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 12px;
+}
+
+.sec2 h3 {
+  color: #fff;
+  font-weight: 500;
+  font-size: 30px;
+  height: 20px;
+  margin-bottom: 13px;
+}
+
+.sec2 img {
+  text-align: center;
+  margin: 5px auto;
+  margin-bottom: 50px;
+}
+
+.b-grey {
+  background-color: #272a31;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 355px;
+  height: 120px;
+  margin-bottom: 10px;
+  padding: 0;
+}
```

Comment:
- [ ] As requested by the previous code reviewer, in the original design there is a hover effect that turns the border color withe. Please 🙏🏽 add it to be similar to the original design for a better UX.